plugin/src/main/java/org/autorefactor/refactoring/ASTBuilder.java

```diff
@@ -857,4 +864,37 @@ public Expression parenthesizeIfNeeded(Expression expr) {
             return expr;
         }
     }
+
+    /**
+     * Builds a new {@link Statement} instance which is basically a newline.
+     *
+     * @return a newline statement
+     */
+    public Statement newlinePlaceholder() {
+        return (Statement) this.refactorings.getRewrite().createStringPlaceholder("\n", ASTNode.EMPTY_STATEMENT);
+    }
+
+    public Modifier modifierProtected() {
+        return getAST().newModifier(ModifierKeyword.PROTECTED_KEYWORD);
+    }
+
+    public List<IExtendedModifier> modifiers(IExtendedModifier... modifiers) {
+        return Arrays.asList(modifiers);
+    }
+
+    public Expression superInvoke(String methodName) {
+        SuperMethodInvocation superMethodInvocation = getAST().newSuperMethodInvocation();

```

Comment:
This is a personal standard, right? IMO clear variable names help read the code and writing the code is quite the same when using autocomplete. (my 2 cents 😄)